packages/synapse-react-client/src/components/UserSearchBoxV2.tsx

```diff
@@ -172,6 +172,7 @@ const UserSearchBoxV2: React.FC<UserSearchBoxProps> = props => {
             gridArea: '1 / 2 / 4 / 4 !important',
           },
         }),
+        menu: base => ({ ...base, zIndex: 2 }),
```

Comment:
Fix minor issue where form labels (with z-index of 1) would appear above the menu options.